kbase-extension/static/kbase/js/api/auth.js

```diff
@@ -277,6 +275,16 @@ define(['bluebird', 'jquery', 'narrativeConfig'], (Promise, $, Config) => {
                 return [url, '/api/', version, callParams.operation].join('');
             })();
 
+            // const options = {
+            //     method: callParams.method,
+            //     headers: {
+            //         Authorization: token,
+            //         'Content-Type': 'application/json',
+            //     },
+
+            // }
+            // return fetch(callString, options);
+
```

Comment:
??

Reply:
The only vestige of jquery is that API call wrapped in a Promise, so I started replacing it with a `fetch` call. Then realized there's some downstream unpacking effects to manage. Then realized that was a silly exercise and wasn't solving anything useful so just put it back and stopped.